client/src/views/Simulation/components/ProvenanceGraph/ProvenanceGraphData.vue

```diff
@@ -1,32 +1,67 @@
 <template>
-  <section class="provenance-graph-data">
-    <h6>{{header}}</h6>
-  </section>
+  <div class="provenance-graph-data" ref="graph" />
 </template>
 
 <script lang="ts">
   import Vue from 'vue';
   import Component from 'vue-class-component';
-  import { Prop } from 'vue-property-decorator';
+  import { Prop, Watch } from 'vue-property-decorator';
+
+  import { expandCollapse } from 'svg-flowgraph';
+
+  import { GraphInterface } from '@/types/typesGraphs';
+
+  import ProvenanceRenderer from '@/graphs/svg/renderers/ProvenanceRenderer';
+  import ELKAdapter from '@/graphs/svg/elk/adapter';
+
+  const DEFAULT_RENDERING_OPTIONS = {
```

Comment:
instead you should import the one from the `EpiRenderer`:
```JS
import { DEFAULT_RENDERING_OPTIONS } from '@/graphs/svg/renderers/EpiRenderer';
```

Reply:
Yeah I think they can look alike, I've imported from `EpiRenderer`